docs/README.md

```diff
@@ -1,87 +1,98 @@
-1. If an access token is specified with **-a:** switch, binaries for the updated packages are downloaded. Content of the packages are used to update the values of properties defined in `PackagePreprocessorDefinitions` tag in .props files
-1. Modified .props file are saved on disk
-## Input
-1. **-d:** Path to the default.config.  Example: 
-    > _-d:c:\default.config_
-1. **-m:** Path to the manifest.json file or to the containing directory.
-   
-   The following two examples use the same manifest file:
-    > _-m:c:\files\manifest.json_
-    > _-m:c:\files\_
-
-   It is also possible to input multiple manifest files. In this case, the files will be processed in order. If the same package is updated from multiple manifest files, version number from the last specified manifest file will be used. To do this, use semicolon (;) to separate the paths as such:
-    > _m:c:\files\manifest.json;c:\just\folder\\;c:\another\fullpath\manifest.json_
-1. **-i:** Path to ignored packages file [**optional**]. Example: 
-    > _-i:c:\files\ignore.txt_
-1. **-idut** Indicates that packages relevant to the .NET Dev UX team are ignored [**optional**].  If **-i:** is also set, the file specified with that option is used, superceding **-idut**.
-    > _-idut_
-1. **-wl:** Path to the whitelist file [**optional**]. If this is specified, only the packages listed in this file will be updated. Each line in the file represents a regex pattern that will potentially match one or more package IDs. This switch will be ignored if the given file is empty.
-    > _-wl:c:\files\whitelist.txt_
-1. **-p**: Path to the directory to search for .props files [**optional**]. If left unspecified, all the .props files under src\SetupPackages in local VS repo will be searched.
-    > _-p:C:\VS\src\SetupPackages_
-1. **-a**: Personal access token to access packages in [VS feed](https://pkgs.dev.azure.com/devdiv/_packaging/VS-CoreXtFeeds/nuget/v3/index.json) [**optional**]. If not specified, props files will not be updated.
-    > _-a:vv8ofhtojf7xuhehrFxq9k5zvvxstrqg2dzsedhlu757_
-1. **-w:** Maximum allowed duration in seconds, excluding downloads [**optional**].  Example: 
-    > _-w:60_
-1. **-ds:** Maximum allowed duration in seconds that can be spent downloading nuget packages [**optional**].  Example: 
-    > _-ds:240_
-1. **-c:** Maximum concurrency of default.config version updates [**optional**].  Example:
-    > _-c:10_
-1. **-bf:** Filter string to specify which builds from the manifest will be inserted.
-    
-    A simple build filter can be written as follows:
-    ```
-    -bf:repo=.*core-setup
-    ```
+# InsertionsClient
+The InsertionsClient, .NET Core command line executable, is a tool that is used to insert a [drop](https://github.com/dotnet/arcade/blob/master/Documentation/Darc.md#gathering-a-build-drop) into Visual Studio by updating _default.config_, _.packageconfig_ and _.props_ files in Visual Studio repo. 

```

Comment:
Consider: The InsertionsClient, .NET Core command line executable, is used to insert a

Reply:
I think the sentence in the PR, as intended, makes this more like a definition. Saying that "... is a tool" gives it a bit more shape, which could be useful to someone who is just learning about Insertions Client.
What exactly do you find unappealing about the proposed sentence?